tests/bot/cogs/test_cogs.py

```diff
@@ -0,0 +1,77 @@
+"""Test suite for general tests which apply to all cogs."""
+
+import importlib
+import pkgutil
+import typing as t
```

Comment:
this tripped me up a bit. is this something everyone does?

Reply:
No, not really. However, I think I stole this idea from some other code bases. Maybe it's a bad habit of mine. The benefit of this is reducing length of annotations and thus increasing readability. If annotations get too long, the function's arguments also have to be broken up, which makes it take up more space. Doing `from typing import x, y, z, a, b, c, d` can work if there are only a couple things but it eventually gets out of control.